### ISSUE TITLE
Question about the content of converting app.js into bundle.js in webpack.config.js in "blob/master/docs/getting-started.md"

### ISSUE THREAD

**Issue by ukalpa** [material-components/material-components-web#2998]
Hi there,

I am trying to use MDC Web with ES2015 and Sass and stucked in Step 3: Webpack with ES2015. [https://github.com/material-components/material-components-web/blob/master/docs/getting-started.md](getting-started.md)

According the description of the document "getting-started.md", I copied the following code to webpack.config.js:
![image](https://user-images.githubusercontent.com/4927815/41883926-13a01b06-7945-11e8-83f8-6fc84d06ceb3.png)

And executed "npm start" but got the error message as follows:
   
> Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
> - configuration[1].module.rules[0].use should be one of these:
>   non-empty string | function | object { loader?, options?, ident?, query? } | function | [non-empty string | function | object { loader?, options?, ident?, query? }]
>   -> Modifiers applied to the module when rule is matched
>   Details:
>    * configuration[1].module.rules[0].use should be a string.
>    * configuration[1].module.rules[0].use should be an instance of function
>    * configuration[1].module.rules[0].use should be an object.

I reviewed my code following the artile step by step but still was unable to find where I did wrong, so I have to change the json data format by simulating the first "css" part as follows. After that, the command"npm start" executed successfully.  Thanks a lot for anyone who could tell me the reason of the issue and hope it is helpful to the person who have the same issue like it.

```
module.exports.push({
    entry: './app.js',
    output: {
        filename: 'javascript-bundle.js'
    },
    module: {
        rules: [{
            test: /\.js$/,
            use: [
                {
                    loader: 'file-loader',
                    options: {
                        name: 'bundle.js',
                    },
                },
                {
                    loader: 'babel-loader',
                    query: {
                        presets: ['es2015']
                    }
                }
            ]
        }]
    }
    // rules: {
    //     loaders: [{
    //         test: /\.js$/,
    //         loader: 'babel-loader',
    //         query: {
    //             presets: ['es2015']
    //         }
    //     }]
    // },
});
```

Reference: package.json
```
{
  "scripts": {
    "start": "webpack-dev-server"
  },
  "devDependencies": {
    "autoprefixer": "^8.6.3",
    "babel-core": "^6.26.3",
    "babel-loader": "^7.1.4",
    "babel-preset-es2015": "^6.24.1",
    "css-loader": "^0.28.11",
    "extract-loader": "^2.0.1",
    "file-loader": "^1.1.11",
    "node-sass": "^4.9.0",
    "postcss-loader": "^2.1.5",
    "sass-loader": "^7.0.3",
    "webpack": "^3.12.0",
    "webpack-dev-server": "^2.11.2"
  },
  "dependencies": {
    "@material/button": "^0.36.1",
    "material-components-web": "^0.36.1"
  }
}
```

**Comment by saulbensach** [comment]
@ukalpa hey man, I had EXACTLY the same problem, and pasting your webpack.config.js it finally worked, so yes, seems to be something wrong...

**Comment by moog16** [comment]
Thanks for finding this. It looks like we recently accepted a doc change related to your issue: https://github.com/material-components/material-components-web/pull/2943. Going to keep this open to track the issue.

**Comment by ukalpa** [comment]
Hi @williamernest 

I checked PR #3013 your submitted and it is working perfectly. So I closed the PR #3012 I submitted before. Thanks a lot for solving the issue and hope the update could be updated to the master asap.

**Comment by williamernest** [comment]
@ukalpa Thanks for submitting the PR. After looking over the docs and your PR we realized we wanted other changes to make it easier to understand what was going on. It's merged now, so let us know if you encounter any other issues.
